### PR TITLE
[MM-16235] System Console: Plugins section is missing for restricted system admin

### PR DESCRIPTION
**Pull request by bradjcoughlin**
#### Summary
When `ExperimentalSettings > RestrictSystemAdmin` is enabled, the system console will show the `Plugins` section in the LHS, but hide the `Enable Plugins` setting.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-16235

### CODE REVIEW

File: components/admin_console/plugin_management/plugin_management.jsx [hunk 1]
```diff
@@ -610,6 +610,33 @@ export default class PluginManagement extends AdminSettings {
...
+                    label={
+                        <FormattedMessage
+                            id='admin.plugins.settings.enable'
+                            defaultMessage='Enable Plugins: '
+                        />
+                    }
+                    helpText={
+                        <FormattedMarkdownMessage
+                            id='admin.plugins.settings.enableDesc'
+                            defaultMessage='When true, enables plugins on your Mattermost server. Use plugins to integrate with third-party systems, extend functionality or customize the user interface of your Mattermost server. See [documentation](https://about.mattermost.com/default-plugin-uploads) to learn more.'
```
Comment (gabrieljackson) [inline components/admin_console/plugin_management/plugin_management.jsx]: nit: I am always a fan of oxford commas, but 0/5 on this.

```suggestion
                            defaultMessage='When true, enables plugins on your Mattermost server. Use plugins to integrate with third-party systems, extend functionality, or customize the user interface of your Mattermost server. See [documentation](https://about.mattermost.com/default-plugin-uploads) to learn more.'
```

**Comment by amyblais** [comment]
@bradjcoughlin Can you help cherry-pick to v5.13?

**Comment by bradjcoughlin** [comment]
@amyblais will do!

**Comment by amyblais** [comment]
@bradjcoughlin Kind reminder ^

**Pull request merged**